web/src/components/Plugin/UI/basic-auth.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import type { FormInstance } from 'antd/es/form';
+import { Form, Input } from 'antd';
+
+type Props = {
+  form: FormInstance;
+  ref?: any;
+};
+
+export const FORM_ITEM_LAYOUT = {
+  labelCol: {
+    span: 4,
+  },
+  wrapperCol: {
+    span: 8
+  },
+};
+
+const BasicAuth: React.FC<Props> = ({ form }) => {
```

Comment:
there is a difference between basic-auth plugin in route and basic-auth plugin in consumer, it would be better to distinguish it according to schematype

Reply:
yep

Reply:
I have changed the logic in this section so that UIForm shows empty when no configuration is required, please review again, thank you. @liuxiran  cc @juzhiyuan 

![image](https://user-images.githubusercontent.com/31329157/114115310-29d32b00-9915-11eb-8692-63047904f55a.png)
![image](https://user-images.githubusercontent.com/31329157/114115418-5d15ba00-9915-11eb-80c0-60927d73030a.png)

Reply:
LGTM, let's wait for CI. thanks